ansible/roles/test/files/ptftests/hash_test.py

```diff
@@ -191,22 +135,25 @@ def check_ipv4_route(self, in_port, sport, dport,
         masked_exp_pkt = Mask(exp_pkt)
         masked_exp_pkt.set_do_not_care_scapy(scapy.Ether, "dst")
 
-        send_packet(self, in_port, pkt)
-        logging.info("Sending packet from port " + str(in_port) + " to " + ip_dst)
+        send_packet(self, src_port, pkt)
+        logging.info("Sending packet from port " + str(src_port) + " to " + ip_dst)
 
         return verify_packet_any_port(self, masked_exp_pkt, dst_port_list)
     #---------------------------------------------------------------------
 
-    def check_ipv6_route(self, in_port, sport, dport,
-                         ip_src, ip_dst, dst_port_list):
+    def check_ipv6_route(self, hash_key, src_port, dst_port_list):
         '''
         @summary: Check IPv6 route works.
-        @param source_port_index: index of port to use for sending packet to switch
-        @param dest_ip_addr: destination IP to build packet with.
+        @param hash_key: hash key to build packet with.
+        @param src_port: index of port to use for sending packet to switch
```

Comment:
can you rename it to in_port, src_port can be easily confused with src-port which is l4 source port.

Reply:
> can you rename it to in_port, src_port can be easily confused with src-port which is l4 source port.

done.